org.eclipse.lyo.oslc4j.plugins/pom.xml

```diff
@@ -0,0 +1,212 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd"
+    xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.eclipse.lyo.tools</groupId>
+        <artifactId>tools-parent</artifactId>
+        <version>4.0.0-SNAPSHOT</version>
+        <relativePath>../pom.xml</relativePath>
+    </parent>
+    <artifactId>org.eclipse.lyo.oslc4j.plugins</artifactId>
+    <packaging>pom</packaging>
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.reficio</groupId>
+                <artifactId>p2-maven-plugin</artifactId>
+                <version>1.3.0</version>
+                <executions>
+                    <execution>
+                        <id>default-cli</id>
+                        <phase>package</phase>
+                        <goals>
+                            <goal>site</goal>
+                        </goals>
+                        <configuration>
+                            <!-- 2020-08-25 Jad: Why am I skipInvalidArtifacts? Otherwise, the script unnecessarily fails for dependencies of type pom. Namely, apache-jena-libs-->
+                            <skipInvalidArtifacts>true</skipInvalidArtifacts>
+                            <categoryFileURL>${basedir}/category.xml</categoryFileURL>
+                            <featureDefinitions>
+                                <feature>
+                                    <id>org.eclipse.lyo.oslc4j.core.plugin.feature</id>
+                                    <version>${project.version}</version>
+                                    <label>Lyo OSLC4J Core</label>
+                                    <providerName>Eclipse Lyo</providerName>
+                                    <description url="https://www.eclipse.org/lyo/"> 
+                                        On OSGI bundle of Lyo OSLC4J Core (including Query and Jena-provider), which is otherwise available under maven.
+                                    </description>
+                                    <copyright url="https://www.eclipse.org/lyo/">
+                                              Copyright (c) 2012 IBM Corporation and others.

```

Comment:
2020 Eclipse contributors and others

this would be better.

Reply:
How about now? Happy?